chromatic/rainbows/actions/conversions.py

```diff
@@ -0,0 +1,79 @@
+from ...imports import *
+
+__all__ = ["to_nparray", "to_df"]
+
+
+def to_nparray(self, timeformat='h'):
```

Comment:
I wonder, could we please default to days instead of hours? I know hours feel a little more intuitive for the timescale of a transit, but as soon as we start talking about fits or comparing to known periods/epochs, we're going to want to think in days.

Reply:
Done!

Reply:
Though I believe that the chromatic rainbow objects are by default in hours (which could be confusing)

Reply:
Ha, fair enough. I'll switch the `SimulatedRainbow` to default to `day` and `micron` for time and wavelength, to try to follow my own suggestion 😆